src/App.js

```diff
@@ -1,7 +1,5 @@
 import './App.css';
-import { BrowserRouter, Route, Routes } from 'react-router-dom';
-import React, { useEffect } from 'react';
-import { useDispatch } from 'react-redux';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
```

Comment:
Can you kindly import useDispatch from react-redux inside the components we are using it.

![image](https://github.com/hydaline-code/space-travelers-hub/assets/103440988/77ccb144-89e4-420b-b549-bd658ab68076)

hopefully, once this error has been fixed our project will run again on the browser